python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -26,3 +26,40 @@ def fit(m, x, y=None):
 
 def fit_transform(m, x):
     m.fit_transform(x)
+
+
+def predict(m, x, y=None):
+    m.predict(x) if y is None else m.predict(x, y)
```

Comment:
`predict(x, y)` seems like a weird use case.. is there a specific usage you have in mind?

Reply:
I am removing `predict(x, y). Thanks for pointing this out